code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -52,7 +60,7 @@ var/global/list/parasites = list() //all currently existing/living guardians
 
 /mob/living/simple_animal/hostile/guardian/proc/setthemename(pickedtheme) //set the guardian's theme to something cool!
 	if(!pickedtheme)
-		pickedtheme = pick("magic", "tech")
+		pickedtheme = pick("magic", "tech", "carp")
```

Comment:
this will make it randomly pick if no theme rather then a preselected theme from the device. you didn't need to touch it.